tasks/npm_post_install.sh

```diff
@@ -21,3 +21,7 @@ sed -i -- "s/_focusedField\.focus()/_focusedField\.trigger('focus')/g" node_modu
 # Github issue: https://github.com/WordPress/gutenberg/issues/32181
 sed -i -- "s/box-shadow: 0 0 0 2px-1px #007cba;/box-shadow: 0 0 0 calc(2px-1px) #007cba;/g" node_modules/@wordpress/block-editor/build-style/style.css
 sed -i -- "s/box-shadow: 0 0 0 var(--wp-admin-border-width-focus)-1px var(--wp-admin-theme-color);/box-shadow: 0 0 0 calc(var(--wp-admin-border-width-focus)-1px) var(--wp-admin-theme-color);/g" node_modules/@wordpress/block-editor/build-style/style.css
```

Comment:
```suggestion
```
Not related to the upgrade but do we still need this? https://github.com/WordPress/gutenberg/pull/32305/files

Reply:
Good catch! I must have missed it when I was updating @wordpress dependencies. I'll remove it.